docs/tutorials/sqlalchemy/index.rst

```diff
@@ -13,8 +13,8 @@ Improving the TODO app with SQLAlchemy
 Install SQLAlchemy
 ==================
 
-To follow this tutorial, you will need SQLAlchemy installed. You can install it with ``pip install sqlalchemy``, or let
-Litestar install it for you by installing the ``sqlalchemy`` extra (e.g., ``pip install 'litestar[standard,sqlalchemy]'``).
+To follow this tutorial, you will need SQLAlchemy installed. You can install it with ``pip install sqlalchemy aiosqlite greenlet``, or let
+Litestar install it for you by installing the ``sqlalchemy`` extra (e.g., ``pip install 'litestar[standard,sqlalchemy] aiosqlite greenlet'``).
```

Comment:
```suggestion
Litestar install it for you by installing the ``sqlalchemy`` extra (e.g., ``pip install 'litestar[standard,sqlalchemy] aiosqlite'``).
```
I've tested this on a clean environment - greenlet is available after `python -m pip install 'litestar[standard,sqlalchemy]'`